telcell/models/simplemodel.py

```diff
@@ -0,0 +1,146 @@
+import datetime
+from typing import List
+from collections import defaultdict
+
+from telcell.data.models import Measurement, Track, MeasurementPair
+
+
+def get_measurement_with_minimum_time_difference(track: Track,
+                                                 timestamp: datetime) \
+        -> Measurement:
+    """
+    Finds the measurement in the track with the smallest time
+    difference compared to the timestamp.
+
+    @param track: A history of measurements for a single device.
+    @param timestamp: The timestamp used to find the closest measurement.
+    @return: The measurement that is the closest to the timestamp
+    """
+    return min(track.measurements, key=lambda m: abs(m.timestamp - timestamp))
+
+
+def make_pair_based_on_time_difference(track: Track,
+                                       measurement: Measurement) \
+        -> MeasurementPair:
+    """
+    Creates a pair based on time difference. The closest measurement in
+    absolute time will be paired to the measurement.
+
+    @param track: A history of measurements for a single device.
+    @param measurement: A single measurement of a device at a certain
+                        place and time.
+    @return: A pair of measurements.
+    """
+    closest_measurement = get_measurement_with_minimum_time_difference(
+        track,
+        measurement.timestamp)
+    return MeasurementPair(closest_measurement, measurement)
+
+
+def pair_measurements_based_on_time(track_a: Track,
+                                    track_b: Track) \
+        -> List[MeasurementPair]:
+    """
+    Pairs two tracks based on the time difference between the measurements.
```

Comment:
Oude comment van Jeroen: Is een functie die paren maakt van elke overgang van een measurement uit `track_a` naar een measurement uit `track_b` en andersom niet een logischere functie? Of hebben we echt deze specifieke logica nodig voor de simpelste methode? Zo goed zit ik namelijk ook niet in deze materie. 

Zie mijn mooie ascii art hieronder voor een visuele toelichting, waarbij `O` steeds een measurement is en alle connecties van A naar B en andersom de paren representeren die je terug krijgt:

```
A:  O----------O---O---O
     \        /     \ /
B:    O---O--O-------O
```

Reply:
Ik dacht dat er vaak een mismatch was tussen de burner telefoon en de telefoon waar we mee moesten vergelijken. Dus vandaar dat we gekeken hebben naar de burner en daar de dichtstbijzijnde paren bij zoeken. Als dit anders is dan kunnen we het altijd veranderen natuurlijk. @wowtor wat is hierbij handig?

Reply:
oude comment van Wouter: We willen alle paren overwegen in het schema van Jeroen. Wat dacht je van een itertools.pairwise(track_a + track_b) en dan een filter op different track?

Reply:
Het schema van Jeroen is eigenlijk wat er nu gebeurt, Behalve dan dat er vanaf B gekeken wordt. Alles van B wordt aan de dichtstbijzijnde measurement van A gekoppeld. Dus wat dat betreft zit het er al in :)